Source/Eto/Forms/Controls/TreeView.cs

```diff
@@ -95,6 +95,7 @@ public TreeViewItemEditEventArgs(ITreeItem item, string label)
 	/// </remarks>
 	/// <copyright>(c) 2014 by Curtis Wensley</copyright>
 	/// <license type="BSD-3">See LICENSE for full terms</license>
+	[Obsolete("TreeView is deprecated, please use TreeGridView instead.")]
```

Comment:
Add "Since 2.3. " to the beginning of that.  Sorry, forgot to mention before. (;